scripts/deploy-debug-ds.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+# Initialization
+SCRIPT_DIR=$(dirname "$0")
+source $SCRIPT_DIR/init-env.sh
+# check if we're using minikube by looking for kube-apiserver-minikube pod
+# if it's minikube, don't install debug partner
+res=`oc get pod -A | grep -i kube-apiserver-minikube`
```

Comment:
To prevent spaces, just return the match:
grep -i
=>
grep -io

Reply:
fixed.